src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -1,16 +1,35 @@
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Fragment } from 'react';
 import Downshift from 'downshift';
 import isEqual from 'lodash.isequal';
 import ListBox from '../ListBox';
 import Checkbox from '../Checkbox';
+import FormLabel from '../FormLabel';
 import Selection from '../../internal/Selection';
 import { sortingPropTypes } from './MultiSelectPropTypes';
 import { defaultItemToString } from './tools/itemToString';
+import { groupedByCategory } from './tools/groupedByCategory';
 import { defaultSortItems, defaultCompareItems } from './tools/sorting';
 import { defaultFilterItems } from '../ComboBox/tools/filter';
 
+const categoryLabel = {
```

Comment:
Same here - these look opinionated, i.e., not 100% portable structure. Let's figure out how we leverage new styles from the IBM crew.